### ISSUE TITLE
Add option to disable prev2/next2 labels

### ISSUE THREAD

**Issue by rmccue** [wojtekmaj/react-calendar#62]
Hi! I'd like to be able to disable the prev2 and next2 buttons entirely. Passing `prev2Label={ null }` and `next2Label={ null }` render buttons without text, but don't actually remove the buttons.

(It's possible to hide these through CSS, but it'd be nice to disable via props as well to avoid even rendering the buttons in the first place.)

**Comment by wojtekmaj** [comment]
Hi there, that is a great idea. It has been implemented and will be live with the next release!

**Comment by fhessenberger** [comment]
Hi! I just noticed that the type definitions for `prev2Label`/`next2Label` don't allow for `null`. Would you like a PR?

**Comment by wojtekmaj** [comment]
Absolutely! Please do. I'm still not knowledgeable enough in TS to be sure I'm not messing up.

**Comment by J0914** [comment]
> .react-calendar__navigation__prev2-button, .react-calendar__navigation__next2-button{ visibility: hidden; }

amazing thank you lol

**Comment by wojtekmaj** [comment]
Passing `null` to prev2Label, next2Label will remove the buttons.
